http/role-tests.http

```diff
@@ -0,0 +1,381 @@
+# @name update-role
+< {%
+  import {wait} from "js/wait";
+  wait(2);
+%}
+PUT {{ZEEBE_REST_ADDRESS_LOCAL}}/v2/roles/{{ROLE_ID}}
+Authorization: Basic {{BASIC_AUTH_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "name": "TestRole Updated",
+  "description": "An updated test role description."
+}
+
+> {%
+  client.test("Group update should be successful", function () {
+    client.assert(response.status === 200, "Response status is not 200");
+    client.assert(response.body.name === "TestRole Updated", "Name was not updated correctly.");
+    client.assert(response.body.description === "An updated test role description.", "Description was not updated correctly.");
+  });
+%}
+
+### Assign User to Role
+# @name assign-user-to-role
+PUT {{ZEEBE_REST_ADDRESS_LOCAL}}/v2/roles/{{ROLE_ID}}/users/{{USERNAME}}
+Authorization: Basic {{BASIC_AUTH_TOKEN}}
+Accept: application/json
+
+> {%
+  client.test("User assignment should be successful", function () {
+    client.assert(response.status === 202, "Response status is not 202");
+  });
+%}
+
+### Search User for Role
+# @name search-user-for-role
+< {%
+  import {wait} from "js/wait";
+  wait(4);
+%}
+POST {{ZEEBE_REST_ADDRESS_LOCAL}}/v2/roles/{{ROLE_ID}}/users/search
+Authorization: Basic {{BASIC_AUTH_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "filter": {
+    "username": "{{USERNAME}}"
+  },
+  "sort": [
+    {
+    "field": "username",
+    "order": "ASC"
+     }
+  ]
+}
+
+> {%
+  client.test("User search should be successful", function () {
+
+    client.assert(response.status === 200, "Response status is not 200");
+    client.assert(response.body.items[0].username === "demo", "User was not found");
+  });
+%}
+
+### Remove User from Role
+# @name remove-user-from-role
+< {%
+  import {wait} from "js/wait";
+  wait(2);
+%}
+DELETE {{ZEEBE_REST_ADDRESS_LOCAL}}/v2/roles/{{ROLE_ID}}/users/{{USERNAME}}
+Authorization: Basic {{BASIC_AUTH_TOKEN}}
+Accept: application/json
+
+> {%
+  client.test("User removal should be successful", function () {
+    client.assert(response.status === 202, "Response status is not 202");
+  });
+%}
+
+### Search Roles
+# @name search-roles
+< {%
+  import {wait} from "js/wait";
+  wait(2);
+%}
+POST {{ZEEBE_REST_ADDRESS_LOCAL}}/v2/roles/search
+Authorization: Basic {{BASIC_AUTH_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+  "filter": {
+    "roleId": "{{ROLE_ID}}"
+  },
+  "sort": [
+    {
+    "field": "name",
+    "order": "ASC"
+     }
+  ]
+}
+
+> {%
+  let expectedRoleId = client.global.get("ROLE_ID");
+
+  client.test("Role search should be successful", function () {
+    client.assert(response.status === 200, "Response status is not 200");
+    client.assert(response.body.items[0].roleId === expectedRoleId, "Role was not found");
+  });
+%}
+
+### Assign Client to Role
+# @name assign-client-to-role
+PUT {{ZEEBE_REST_ADDRESS_LOCAL}}/v2/roles/{{ROLE_ID}}/clients/tasklist
```

Comment:
:thought_balloon: It probably makes sense to make the client id a variable too, just like `USERNAME`. Also, `tasklist` could be confusing as it is also an application.